File: programmers/python 3/12930.py
# 문제 링크: https://programmers.co.kr/learn/courses/30/lessons/12930

def solution(s):
    def convert(s: str) -> str:
        return ''.join(
            s[i].lower() if i % 2 else s[i].upper()
            for i in range(len(s))
        )

    return ' '.join(map(convert, s.split(' ')))

File: programmers/swift/12930.swift
// 문제 링크: https://programmers.co.kr/learn/courses/30/lessons/12930

func solution(_ s:String) -> String {
    return s
        .components(separatedBy: " ")
        .map { $0
            .enumerated()
            .map { $0 % 2 == 0 ? $1.uppercased() : $1.lowercased() }
            .joined()
        }
        .joined(separator: " ")
}